docs/source/AdministratorGuide/HowTo/pilotsWithTokens.rst

```diff
@@ -0,0 +1,73 @@
+.. _pilots-with-tokens:
+
+=====================================
+Submitting pilots to CEs using tokens
+=====================================
+
+
+This guide outlines the process of setting up DIRAC to submit pilots using access tokens obtained via a `client_credentials` flow from a token provider.
+
+Setting up an `IdProvider`
+--------------------------
+
+- Set up an OAuth2 client in the token provider and obtain a `client_id` and a `client_secret`.
+
+    .. warning:: The client credentials obtained are confidential, store them in a secure place.
+       Any malicious user able to get access to them would be able to generate access tokens on your behalf.
+       To avoid any major issue, we recommend you to only grant essential privileges to the client (`compute` scopes).
+
+- Add the client credentials in your local `dirac.cfg` configuration such as:
+
+    .. code-block:: guess
+
+      Resources
+      {
+        IdProviders
+        {
+          <IdProvider name>
+          {
+            client_id = <client_id>
+            client_secret = <client_secret>
+          }
+        }
+      }
+
+- Then in your global configuration, add the following section to set up an `IdProvider` interface:
+
+    .. code-block:: guess
+
+      Resources
+      {
+        IdProviders
+        {
+          <IdProvider name>
+          {
+            issuer = <OIDC provider issuer URL>
+          }
+        }
+      }
+
+- Connect the OIDC provider to a specific VO by adding `<IdProvider name>` in `/Registry/VO/<VO name>`.
```

Comment:
```suggestion
- Connect the OIDC provider to a specific VO by adding the respective `<IdProvider name>` section under `/Registry/VO/<VO name>`.
```
If that is what is meant? Does it go under a `IdProviders` section as well?

Reply:
No, I expanded that part with an example to make it more clear.

Reply:
Thanks, that is clear now.